src/DOM.js

```diff
@@ -2199,7 +2199,9 @@ class HTMLIFrameElement extends HTMLSrcableElement {
                 const parentWindow = this.ownerDocument.defaultView;
                 const options = parentWindow[symbols.optionsSymbol];
 
-                url = _normalizeUrl(url, options.baseUrl);
+                url = _normalizeUrl(res.url, options.baseUrl);
+                // passively update .src with the final url (which can differ in case of redirects)
+                _setAttributeRaw(this, 'src', url, false);
```

Comment:
Is that necessary/how `<iframe>` works in general? Or is the redirect transparent?

Reply:
probably not necessary -- want me to update to be a one liner?

note that in browsers `iframe.src` _does_ reflect the final url after redirects, but only when accessed from allowable origins; otherwise it maintains the original input url.   (at least in Firefox and Chrome)

Reply:
Interesting, thanks for the the investigation -- makes sense with the cross-origin behavior.

If it doesn't change the behavior of the fix, I'd prefer to keep the changes minimal (i.e. https://github.com/exokitxr/exokit/pull/1408#discussion-diff-319777292R2202O).